doc/development/guide/pullrequests.rst

```diff
@@ -0,0 +1,66 @@
+Submitting a pull request
+=========================
```

Comment:
While on this page, (similar to the `Software tests` page) the `Contents` section appears on the right empty. Is there a way to either add a small content section/remove it easily?

Reply:
I'm not sure what's the best solution here. What could be done is to restructure the `Software tests` and `Submitting a pull request` pages to have subheaders (which will then show up in the `Contents` section).

Reply:
I see yes! That sounds like a solution indeed. I'd assume that for `Software tests` that might cut up the content a bit too much (given that it's a pretty short page)? Overall I'm happy with, either way, was just curious if there was a quick fix for this :)

Reply:
@antalszava I've added subheaders to some of the pages now. Let me know what you think.